parts/iaasoutputs.t

```diff
@@ -25,4 +25,13 @@
     "primaryAvailabilitySetName": {
         "type": "string",
         "value": "[variables('primaryAvailabilitySetName')]"
+    },
+    "primaryScaleSetName": {
+        "type": "string",
+        "value": "[variables('primaryScaleSetName')]"
+    },
+    "vmType": {
```

Comment:
we really dont need this

Reply:
This change is originally target to pass the cloud config settings of vmType and primaryScaleSetName to AKS (in the way of exposing it to ARM output).

I will see if we can build these strings in AKS for its cloud provider config.